default_shardings/llama-3.yaml

```diff
@@ -0,0 +1,28 @@
+
+# Sharding config for llama-3
```

Comment:
There is no different between llama-2 sharding vs llama3 sharding. I feel we should keep only 1 sharding, it's easy to maintain and manage.